tests/Dfc.CourseDirectory.WebV2.Tests/FeatureTests/EditVenue/DetailsTests.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Dfc.CourseDirectory.WebV2.Features.EditVenue;
+using FormFlow;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace Dfc.CourseDirectory.WebV2.Tests.FeatureTests.EditVenue
+{
+    public class DetailsTests : MvcTestBase
+    {
+        public DetailsTests(CourseDirectoryApplicationFactory factory)
+            : base(factory)
+        {
+        }
+
+        [Theory]
+        [InlineData(TestUserType.ProviderSuperUser)]
+        [InlineData(TestUserType.ProviderUser)]
+        public async Task Get_UserCannotAccessVenue_ReturnsForbidden(TestUserType userType)
+        {
+            // Arrange
+            var providerId = await TestData.CreateProvider();
+            var venueId = await TestData.CreateVenue(providerId);
+
+            var anotherProviderId = await TestData.CreateProvider(ukprn: 67890);
+
+            var request = new HttpRequestMessage(HttpMethod.Get, $"venues/{venueId}");
+
+            await User.AsTestUser(userType, anotherProviderId);
+
+            // Act
+            var response = await HttpClient.SendAsync(request);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.Forbidden, response.StatusCode);
+        }
+
+        [Fact]
+        public async Task Get_VenueDoesNotExist_ReturnsNotFound()
+        {
+            // Arrange
+            var venueId = Guid.NewGuid();
+
+            var request = new HttpRequestMessage(HttpMethod.Get, $"venues/{venueId}");
+
+            // Act
+            var response = await HttpClient.SendAsync(request);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
+        }
+
+        [Fact]
+        public async Task Get_ValidRequestNoExistingFormFlowInstance_RendersExpectedOutputFromDatabase()
+        {
+            // Arrange
+            var providerId = await TestData.CreateProvider();
+            var venueId = await TestData.CreateVenue(
+                providerId,
+                venueName: "Test Venue",
+                email: "test-venue@provider.com",
```

Comment:
I shy away from using real domain names. Suggest using `example.com` which is reserved.